core/game_object.lua

```diff
@@ -2302,10 +2310,6 @@ function loadAPIs()
             SMODS.process_loc_text(G.localization.misc.labels, self.key:sub(3), self.loc_txt, 'label')
             SMODS.Edition.super.process_loc_text(self)
         end,
-        -- apply_modifier = true when G.GAME.edition_rate is to be applied
-        get_weight = function(self, apply_modifier)
```

Comment:
Why was get_weight removed? also get_weight remains a bad name, sorry. to be honest, letting weight itself be either a number or a function might be better

Reply:
I have no idea how this has been removed. I've renamed to `modified_weight` instead, but you need to be able to access the base weight and the modified weight to let `poll_edition` function correctly.